account/src/main/java/com/caldevsupplychain/account/config/SwaggerConfig.java

```diff
@@ -16,7 +16,7 @@
 	public Docket api() {
 		return new Docket(DocumentationType.SWAGGER_2)
 				.select()
-				.apis(RequestHandlerSelectors.any())
+				.apis(RequestHandlerSelectors.basePackage("com.caldevsupplychain.account.controller"))
```

Comment:
Can prob use `com.caldevsupplychain`, which might include order and other packages

Reply:
fixed

Reply:
fixed in JWT merged PR